docs/data-access/data-download.md

```diff
@@ -0,0 +1,89 @@
+---
+id: data-download
+title: How to Download Data
+---
+
+The ARGO Data Platform uses the score-cient as a file download manager. Score facilitates the transfer of data with resumable download and built in BAM/CRAM slicing to make data download transfer fast and smooth.
+
+Please note:
+
+- Downloads are done in part and can be paused/resumed as needed
+- The score-client will automatically resume downloads if interrupted or paused briefly
+
+## Searching for Files
+
+Use the Platform [File Repository](https://platform.icgc-argo.org/repository) to search for a file set of interest. Query results can be narrowed down by selecting specific file values in the facet filters on the left side of the page. Once you have a file set identified, click `Download > File Manifest` on the top right of the table to download a TSV file manifest.
+
+The file manifest will contain a listing of the files that matched your query, along with some additional metatdata to assist in file identification. This file manifest will be used by the score-client as a list of files to download.
```

Comment:
suggestion:
The file manifest contains a listing of the files that match your search query, along with some additional metadata to assist in file identification. The file manifest will be used by the score-client to identify the list of files to download.